compile_commands.json doesn't pass .cpp input file or .o output file as arguments

This causes the following output from clang-tidy:
```
== clang-tidy /home/tav/frc/wpilib/allwpilib/hal/src/main/native/sim/AnalogTrigger.cpp ==
error: no input files [clang-diagnostic-error]
error: unable to handle compilation, expected exactly one compiler job in '' [clang-diagnostic-error]
```

--- a/build.gradle
+++ b/build.gradle
@@ -6,7 +6,7 @@ plugins {
 }
 
 group 'edu.wpi.first'
-version '2.0.0'
+version '2.1.0'
 
 if (project.hasProperty('publishVersion')) {
     version = project.publishVersion

--- a/src/main/java/edu/wpi/first/vscode/compilecommands/BinaryCompileCommandsTask.java
+++ b/src/main/java/edu/wpi/first/vscode/compilecommands/BinaryCompileCommandsTask.java
@@ -45,10 +45,13 @@ public abstract class BinaryCompileCommandsTask extends DefaultTask {
             CompileCommand c = new CompileCommand();
             c.directory = getProject().getProjectDir().getAbsolutePath();
             c.file = f.getAbsolutePath();
+            c.output = spec.getNamingScheme().map(f).getAbsolutePath();
             c.arguments = new ArrayList<>(args.size() + 1);
             c.arguments.add(spec.getCompileTool().getAbsolutePath());
             c.arguments.addAll(args);
-            c.output = spec.getNamingScheme().map(f).getAbsolutePath();
+            c.arguments.add("-o");
+            c.arguments.add(c.output);
+            c.arguments.add(c.file);
             commands.add(c);
         }
 

